charts/refinery/templates/deployment-marker-job.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1
```

Comment:
We should wrap all these contents in a check for `{{- if .Values.deployMarker.enabled }}`